include/core/parameters.h

```diff
@@ -31,12 +31,15 @@
 #ifndef lethe_parameters_h
 #define lethe_parameters_h
 
-#include <core/ib_particle.h>
-
 #include <deal.II/base/conditional_ostream.h>
 #include <deal.II/base/parameter_handler.h>
 #include <deal.II/base/parsed_function.h>
 
+//#include <deal.II/lac/vector.h>
+
+#include <core/ib_particle.h>
+#include <core/multiphysics.h>
+
```

Comment:
Can you remove commented include?
Also, Lethe includes should be above the deal.II includes generally. The automatic indentation should take care of that.